### PR TITLE
Fix servicenow image upload

### PR DESCRIPTION
**Pull request by ostolero**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-18949

## Description
Added content-type to the request depending on the file type.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py [hunk 1]
```diff
@@ -761,6 +762,22 @@ def send_request(self, path: str, method: str = 'GET', body: dict = None, params
...
+
+        Returns:
+            the content type - image with right type for images , and general for other types..
+        """
+        file_type = None
+        if not file_name:
+            demisto.log("file name was not supllied, uploading with general type")
+        else:
+            file_type, _ = mimetypes.guess_type(file_name)
+        return file_type or '*/*'
```
Comment (JudahSchwartz) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py:779]:
```suggestion
        if not file_name:
            demisto.log("file name was not supllied, uploading with general type")
            return '*/*'
        else:
            file_type, _ = mimetypes.guess_type(file_name)
            return file_type
```
Seems clearer to me

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py [hunk 2]
```diff
@@ -1832,6 +1834,21 @@ def ticket_fields_mocker(*args, **kwargs):
...
+                          ('123.jpeg', 'image/jpeg'),
+                          ('123.pdf', 'application/pdf'),
+                          ('123', '*/*')])
+def test_upload_file_types(file_name, expected):
+    client = Client(server_url='https://server_url.com/', sc_server_url='sc_server_url',
+                    cr_server_url='cr_server_url', username='username',
+                    password='password', verify=False, fetch_time='fetch_time',
+                    sysparm_query='sysparm_query', sysparm_limit=10, timestamp_field='opened_at',
+                    get_attachments=False, incident_name='description', ticket_type='incident')
+    assert client.get_content_type(file_name) == expected
```
Comment (JudahSchwartz) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py:1849]: nice test

File: Packs/ServiceNow/ReleaseNotes/2_5_6.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### ServiceNow v2
+- Fixed an issue in **servicenow-upload-file** where images were uploded like files and missed preview.
```
Comment (JudahSchwartz) [inline Packs/ServiceNow/ReleaseNotes/2_5_6.md:4]:
```suggestion
- Fixed an issue in **servicenow-upload-file** where images were missing preview when uploaded.
```

**Review by JudahSchwartz — APPROVED** [review]
nice!

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py (hunk 1 above)
Comment (ostolero) [reply Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py:779]: `mimetypes.guess_type` can return `None` if the type was not identified. 
Any better way to check for this?

**Review by JudahSchwartz — APPROVED** [review]
nice

**Pull request merged**